<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFunctionCheckAndRedioButton</class>
 <widget class="QDialog" name="DialogFunctionCheckAndRedioButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>第一组 复选框 复选</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>香蕉</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/xiangjiao.png</normaloff>:/images/xiangjiao.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>苹果</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/pingguo.png</normaloff>:/images/pingguo.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>草莓</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/caomei.png</normaloff>:/images/caomei.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>第二组 复选框 单选</string>
   </property>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>香蕉</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/xiangjiao.png</normaloff>:/images/xiangjiao.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>苹果</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/pingguo.png</normaloff>:/images/pingguo.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>草莓</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/caomei.png</normaloff>:/images/caomei.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>第三组 单选框 复选</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>苹果</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/pingguo.png</normaloff>:/images/pingguo.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>香蕉</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/xiangjiao.png</normaloff>:/images/xiangjiao.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>草莓</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/caomei.png</normaloff>:/images/caomei.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>第四组 单选框 单选</string>
   </property>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>苹果</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/pingguo.png</normaloff>:/images/pingguo.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>香蕉</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/xiangjiao.png</normaloff>:/images/xiangjiao.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>草莓</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/caomei.png</normaloff>:/images/caomei.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>第六组 复选框 三态</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>犹豫中</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/idea.png</normaloff>:/images/idea.png</iconset>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未关注</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/weiguanzhu.png</normaloff>:/images/weiguanzhu.png</iconset>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>已关注</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/guanzhu.png</normaloff>:/images/guanzhu.png</iconset>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>第五组 混合框 单选</string>
   </property>
   <widget class="QRadioButton" name="radioButton_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>香蕉</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/xiangjiao.png</normaloff>:/images/xiangjiao.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_15">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>草莓</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/caomei.png</normaloff>:/images/caomei.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>苹果</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/pingguo.png</normaloff>:/images/pingguo.png</iconset>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
